src/utils/toornament.ts

```diff
@@ -82,11 +82,12 @@ export const fetchParticipantsDiscordIds = async (toornamentId: string) => {
 
     if (!participant.lineup) {
       if (!participant.custom_fields || !participant.custom_fields.discord) {
-        throw new Error(`${participant.name} don't have a discord id`);
+        logger.warn(`${participant.name} don't have a discord id`);
+        return { name: participant.name };
       }
       return {
         name: participant.name,
-        discordIds: [participant.custom_fields.discord],
+        discordIds: [participant.custom_fields.discord.replace(' #', '#')],
```

Comment:
Wut ? C'est dans quel cas qu'il y a un espace ? Same pour ceux du dessous.

Reply:
Juste pour les gens qui mettent genre "Valentin #7239" ^^' (y'en a pas mal)

Reply:
Je suis chaud du coup pour remplacer ça par
```suggestion
        discordIds: [participant.custom_fields.discord.replace(/ +#/, '#')],
```
au cas où le mec ajoute plus d'espace et englober le tout dans une fonction utils

Reply:
done